src/argilla/server/policies.py

```diff
@@ -369,6 +369,16 @@ async def is_allowed(actor: User) -> bool:
 
 
 class RecordPolicyV1:
+    @classmethod
+    def update(cls, record: Record) -> PolicyAction:
+        async def is_allowed(actor: User) -> bool:
+            return actor.is_owner or (
+                actor.is_admin
+                or await _exists_workspace_user_by_user_and_workspace_id(actor, record.dataset.workspace_id)
```

Comment:
This should be `and` instead of `or` right?

```suggestion
                and await _exists_workspace_user_by_user_and_workspace_id(actor, record.dataset.workspace_id)
```

Reply:
yes!